replica/database.hh

```diff
@@ -1416,9 +1416,12 @@ public:
 private:
     replica::cf_stats _cf_stats;
     static constexpr size_t max_count_concurrent_reads{100};
+    static constexpr size_t max_count_concurrent_view_update_reads{100};
```

Comment:
I thought from the commit message you wanted this to be **half** the regular writes?
Or, perhaps when you said "half" you just meant that you have both regular reads and view read-before-writes, since both have the **same** semaphore size, they get roughly equal work done, so each gets **half**? If this is the answer, perhaps this can be explained more clearly.

Reply:
In the current version this value was actually unused because we weren't performing admission of the view update reads anyway. In the rebase where I add the read admission I set it to 50 so it should now be consistent with the description